data/map planets.txt

```diff
@@ -1496,6 +1498,7 @@ planet Firefall
 	attributes hai station uninhabited
 	landscape land/station36
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	Dwarfed by the nearby Quarg ringworld, Firefall is nevertheless capable of holding countless Hai. The station, like most others in Hai space, was once used to house those who migrated from the Hai's crumbling galactic empire without placing undue strain on their planets' natural resources. As their society adjusted to living within their systems behind the wormhole, residents moved planetside, and the once-massive population of this station slowly dwindled. Now it is quite literally a shell of its former self, having been emptied out of all non-essential equipment long ago.`
```

Comment:
```suggestion
	description `	Dwarfed by the nearby Quarg ringworld, Firefall is nevertheless capable of holding countless Hai. The station, like others of its type in Hai space, was once used to house those who migrated from the Hai's crumbling galactic empire. Now it is quite literally a shell of its former self, having been emptied out of all non-essential equipment long ago.`
```
Too much exposition, but otherwise nice.